pandas/core/methods/to_dict.py

```diff
@@ -160,7 +171,11 @@ def to_dict(
         return into_c(
             (
                 k,
-                list(map(maybe_box_native, v.to_numpy().tolist()))
+                list(
+                    map(
+                        maybe_box_native, v.to_numpy(na_value=box_na_values[i]).tolist()
```

Comment:
We can't just go to `v.tolist()` here?

Reply:
No that breaks something but don't remember which test